src/SDKs/RecoveryServices/Management.RecoveryServices/Customizations/RecoveryServicesClient.cs

```diff
@@ -39,9 +39,14 @@ public RecoveryServicesClient(System.Uri baseUri, ServiceClientCredentials crede
             DisableDispose = disableDispose;
         }
 
+        public void SetHttpClient(HttpClient client)
```

Comment:
FYI
@sonathan the new version 2.3.9 has an overload that will allow you to control disposing of the provided client